examples/typescript/README.md

```diff
@@ -0,0 +1,728 @@
+/******/ 	// mode & 8|1: behave like require
+/******/ 	__webpack_require__.t = function(value, mode) {
+/******/ 		if(mode & 1) value = __webpack_require__(value);
+/******/ 		if(mode & 8) return value;
+/******/ 		if((mode & 4) && typeof value === 'object' && value && value.__esModule) return value;
+/******/ 		var ns = Object.create(null);
+/******/ 		__webpack_require__.r(ns);
+/******/ 		Object.defineProperty(ns, 'default', { enumerable: true, value: value });
+/******/ 		if(mode & 2 && typeof value != 'string') for(var key in value) __webpack_require__.d(ns, key, function(key) { return value[key]; }.bind(null, key));
+/******/ 		return ns;
+/******/ 	};
+/******/
+/******/ 	// getDefaultExport function for compatibility with non-harmony modules
+/******/ 	__webpack_require__.n = function(module) {
+/******/ 		var getter = module && module.__esModule ?
+/******/ 			function getDefault() { return module['default']; } :
+/******/ 			function getModuleExports() { return module; };
+/******/ 		__webpack_require__.d(getter, 'a', getter);
+/******/ 		return getter;
+/******/ 	};
+/******/
+/******/ 	// Object.prototype.hasOwnProperty.call
+/******/ 	__webpack_require__.o = function(object, property) { return Object.prototype.hasOwnProperty.call(object, property); };
+/******/
+/******/ 	// __webpack_public_path__
+/******/ 	__webpack_require__.p = "dist/";
+/******/
+/******/
+/******/ 	// Load entry module and return exports
+/******/ 	return __webpack_require__(__webpack_require__.s = 0);
+/******/ })
+/************************************************************************/
+```
+
+</details>
+
+```javascript
+/******/ ([
+/* 0 */
+/*!********************!*\
+  !*** ./example.js ***!
+  \********************/
+/*! no static exports found */
+/***/ (function(module, exports, __webpack_require__) {
+
+console.log(__webpack_require__(/*! ./index */ 1));
+
+
+/***/ }),
+/* 1 */
+/*!******************!*\
+  !*** ./index.ts ***!
+  \******************/
+/*! no static exports found */
+/***/ (function(module, exports) {
+
+throw new Error("Module build failed (from (webpack)/node_modules/ts-loader/index.js):\nError: TypeScript emitted no output for ./index.ts.\n    at makeSourceMapAndFinish ((webpack)/node_modules/ts-loader/dist/index.js:78:15)\n    at successLoader ((webpack)/node_modules/ts-loader/dist/index.js:68:9)\n    at Object.loader ((webpack)/node_modules/ts-loader/dist/index.js:22:12)");
+
+/***/ })
+/******/ ]);
+```
+
+# Info
+
+## Unoptimized
+
+```
+Hash: 0a1b2c3d4e5f6a7b8c9d
+Version: webpack 4.34.0
+    Asset      Size  Chunks             Chunk Names
+output.js  4.47 KiB       0  [emitted]  main
+Entrypoint main = output.js
+chunk    {0} output.js (main) 610 bytes [entry] [rendered]
+    > ./example.js main
+ [0] ./example.js 33 bytes {0} [built]
+     single entry ./example.js  main
+ [1] ./index.ts 577 bytes {0} [built] [failed] [1 error]
+     cjs require ./index [0] ./example.js 1:12-30
+
+ERROR in ./index.ts
```

Comment:
TS has may errors.

Reply:
It corresponds by transpileOnly of options.